play-services-fido/core/src/main/kotlin/org/microg/gms/fido/core/transport/screenlock/ScreenLockCredentialStore.kt

```diff
@@ -164,10 +164,11 @@ class ScreenLockCredentialStore(val context: Context) : SQLiteOpenHelper(context
     companion object {
         const val TAG = "FidoLockStore"
 
-        const val DATABASE_VERSION = 1
+        const val DATABASE_VERSION = 2
```

Comment:
I know your patch isn't merged upstream yet, but I'm already using your patch; so if you want to avoid the migration, can you at least keep version = 2 ?